store/authStore.js

```diff
@@ -16,7 +17,7 @@ export const authGetters = {
 
 export const authActions = {
   async refreshUser({ commit }) {
-    console.log("refreshing current user.");
+    logger.info("Refreshing current user.");
```

Comment:
I think this one can actually be `.debug`

Reply:
Done ✅